app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
```

Comment:
I think using open by <%- need close by -%> and same for if and end

Reply:
I think it's not really need to end with -%>. From https://api.rubyonrails.org/classes/ActionView/Base.html 
![image](https://github.com/user-attachments/assets/d332edec-608a-4891-9206-6ee073f0fce7)

Reply:
And from other document I found that:

    <%- — If the tag is indented, trim the indentation.
    -%> — If the tag ends a line, trim the following line break.